templates/search_results.hbs

```diff
@@ -43,7 +43,7 @@
       {{/if}}
       {{#if type_filters}}
         <section class="filters-in-section collapsible-sidebar" aria-expanded="false">
-          <button type="button" class="collapsible-sidebar-toggle" aria-expanded="false">
+          <button aria-label="{{t 'search_result_type_menu'}}" type="button" class="collapsible-sidebar-toggle" aria-expanded="false">
```

Comment:
Nitpick: The translation key might be misleading since this is not a menu, but a button. Consider naming this `search_result_type_filters_expand_button`

Reply:
The aria-label will be "Search Result Type Menu" as suggested by the reporter in the task https://zendesk.atlassian.net/jira/software/c/projects/PRODA11Y/boards/3358?modal=detail&selectedIssue=PRODA11Y-219, so I think this should be fine?

Reply:
What is the value of this string? To make them more user friendly we often try to name them closely to how they read like in english, e.g. `add_to_conversation` -> `Add to conversation`

Reply:
`Search Result Type Menu`, `Search Result Source Menu` and `Search Result Subfilters Menu`, so that's in line with what you said